src/pages/feed/index.vue

```diff
@@ -460,20 +457,18 @@ export default {
       this.lazyGetUserInfoByAddress(this.getAddress);
     },
     async batchFetchMemo() {
-      const consistentPendingMemoFetchList = this.pendingMemoFetchList;
+      this.allowFetch = false;
       const currentEventToFetch = Math.min(
         this.eventsToFetch,
-        consistentPendingMemoFetchList.length
-      );
-      const fetchList = consistentPendingMemoFetchList.slice(
-        0,
-        currentEventToFetch
+        this.pendingMemoFetchList.length
       );
+      const fetchList = this.pendingMemoFetchList.slice(0, currentEventToFetch);
 
       const fetchPromises = fetchList.map(event =>
         this.lazyFetchEventsMemo(event)
       );
       await Promise.all(fetchPromises);
+      this.allowFetch = true;
```

Comment:
Should we wrap it in `try catch finally` and set `allowFetch = true` in finally?